lib/utils.js

```diff
@@ -45,9 +45,19 @@ export function replaceRanges(str, ranges, value) {
 	for (let i = ranges.length - 1; i >= 0; i--) {
 		const r = ranges[i];
 
+        let offset = 0;
+        let offsetLength = 0;
+        if (str.substr(r[0] + r[1], 1) === '@'){
+            const matches = str.substr(r[0] + r[1] + 1).match(/^(\d+)/);
```

Comment:
There’s also another option to reverse direction of numbering: https://docs.emmet.io/abbreviations/syntax/#changing-numbering-base-and-direction

But will approve for now